website/scripts/syncExampleDocs.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const path = require('path');
+
+const rootExamples = '../../examples';
+const webDocs = '../docs/examples';
```

Comment:
The current build is failed.
https://app.netlify.com/sites/jest-preview-library/deploys/62bc8bb04f81ca0009cc8374

Remember the build settings
- the base directory is `website`
- the command to build is `npm run build`
![image](https://user-images.githubusercontent.com/8603085/176502191-0d3375ef-37ee-4e82-b2e0-39b32ded4b09.png)

You might need to adjust the script.